clinica/iotools/converters/genfi_to_bids/genfi_to_bids.py

```diff
@@ -45,9 +50,19 @@ def convert_images(
         write_bids,
     )
 
+    # check that if a clinical tsv is given, a path to the clinical data is given as well
+    if path_to_clinical_tsv and not path_to_clinical:
+        raise ValueError("Missing a clinical_data_path.")
```

Comment:
I can imagine users getting stuck here when providing only `path_to_clinical_tsv` but not `path_to_clinical` since these concepts are quite close for the distracted reader of the documentation.

I would be very explicit to ease debugging. Maybe something like:

```python
raise ValueError(
    "The Genfi2BIDS converter is unable to convert the clinical data because "
    "the path to these data was not provided while a TSV file with additional "
    f"data was given ({path_to_clinical_tsv}). You can either use the appropriate "
    "option from the clinica command line interface to provide the missing path, "
    "or chose to not convert clinical data at all."
)
```

Reply:
Sounds good